packages/story-editor/src/components/canvas/karma/backgroundCopyPaste.karma.js

```diff
@@ -55,16 +55,14 @@ describe('Background Copy Paste integration', () => {
 
     // Verify setup - 1 element on each page in the right color
     await gotoPage(1);
-    expect(await getPageArea()).toHaveStyle(
-      'background-color',
-      'rgb(255, 0, 0)'
-    );
+    await waitFor(() => {
+      expect(getPageArea()).toHaveStyle('background-color', 'rgb(255, 0, 0)');
```

Comment:
Here, since `getPageArea` is a `querySelector` which could return `null`, the `waitFor` won't repoll if the first return happens to be `null` since that is a valid option. If we want to wait until it isn't `null` we would have to manually throw an error within this `waitFor` ie.
```
 await waitFor(() => {
  const pageArea = getPageArea();
  if(!pageArea) {
    throw new Error('not ready!')
  }
    expect(getPageArea()).toHaveStyle('background-color', 'rgb(0, 255, 0)');
});
```

Reply:
Thanks for pointing that out, Brooke! Since there are still changes coming in this PR, I'll leave the tests for the last but I'll definitely go through everything making sure `waitFor` is used correctly and double check the usage of `getByRole` => `findByRole`.